packages/manager/apps/container/src/container/common/popover.module.scss

```diff
@@ -0,0 +1,27 @@
+@import '@ovh-ux/ui-kit/dist/scss/tokens/_colors';
+@import '@ovh-ux/ui-kit/dist/scss/tokens/_fonts';
+@import '@ovh-ux/ui-kit/dist/scss/tokens/_globals';
+
+.popover {
+  position: absolute !important;
```

Comment:
All those `!important` are mandatory ? Using a more specific selector doesn't do the trick ?

Reply:
Those `!important` are there because we still use bs popover styles with oui-popover styles and a lot of styles are in conflict... :-(
The solution I see to avoid that is to remove those styles from the scss module and to create a scss file with popover styles wrapped into a parent class:
```scss
.container {
  .popover {
    ...
  }
}
```